demo/server/server.ts

```diff
@@ -0,0 +1,210 @@
+// Create Workflows
+// ============================================
+
+/**
+ * Simple greeting workflow
+ * Input: string (name)
+ * Output: string (greeting message)
+ */
+const greetingWorkflow = createWorkflow();
+greetingWorkflow.handle([greetStartEvent], (_context, event) => {
+  const name = event.data || "World";
+  return greetStopEvent.with(`Hello, ${name}! Welcome to the Workflow Server.`);
+});
+
+/**
+ * Calculator workflow
+ * Input: { a: number, b: number, op: string }
+ * Output: { result: number }
+ */
+const calculatorWorkflow = createWorkflow();
+calculatorWorkflow.handle([calcInputEvent], (_context, event) => {
+  const { a, b, op } = event.data;
+  let result: number;
+
+  switch (op) {
+    case "add":
+      result = a + b;
+      break;
+    case "subtract":
+      result = a - b;
+      break;
+    case "multiply":
+      result = a * b;
+      break;
+    case "divide":
+      if (b === 0) {
+        throw new Error("Division by zero");
+      }
+      result = a / b;
+      break;
+    default:
+      throw new Error(`Unknown operation: ${op}`);
+  }
+
+  return calcOutputEvent.with({ result });
+});
+
+/**
+ * Echo workflow - returns whatever input it receives
+ * Useful for testing and debugging
+ */
+const echoWorkflow = createWorkflow();
+echoWorkflow.handle([echoStartEvent], (_context, event) => {
+  return echoStopEvent.with(event.data);
+});
+
+// ============================================
+// Create and Configure Server
+// ============================================
+
+const server = new WorkflowServer();
+
+// Register workflows
+server.registerWorkflow("greeting", {
+  workflow: greetingWorkflow,
+  startEvent: greetStartEvent,
+  stopEvent: greetStopEvent,
+});
+
+server.registerWorkflow("calculator", {
+  workflow: calculatorWorkflow,
+  startEvent: calcInputEvent,
+  stopEvent: calcOutputEvent,
+});
+
+server.registerWorkflow("echo", {
+  workflow: echoWorkflow,
+  startEvent: echoStartEvent,
+  stopEvent: echoStopEvent,
+});
+
+// ============================================
+// Start Fastify Server
+// ============================================
+
+const app = Fastify({
+  logger: true,
+});
+
+// Register OpenAPI/Swagger documentation
+await app.register(swagger, {
+  openapi: {
+    info: {
+      title: "Workflow Server Demo",
+      description:
+        "A demo server showcasing @llamaindex/workflow-server capabilities",
+      version: "1.0.0",
+    },
+  },
+});
+
+await app.register(swaggerUi, {
+  routePrefix: "/documentation",
+});
+
+// Register the workflow server plugin
+await app.register(server.plugin());
+
+// Add a custom root route
+app.get("/", async () => {
+  return {
+    message: "Workflow Server Demo",
+    documentation: "/documentation",
+    endpoints: {
+      health: "GET /health",
+      listWorkflows: "GET /workflows",
+      runWorkflow: "POST /workflows/:name/run",
+    },
+    registeredWorkflows: server.getWorkflowNames(),
```

Comment:
are there after names?
```suggestion
    registeredWorkflows: server.names(),
```